RecoLocalCalo/HGCalRecProducers/src/classes.h

```diff
@@ -0,0 +1,11 @@
+#include "DataFormats/Common/interface/Wrapper.h"
+
+// Add includes for your classes here
+#include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
+
+namespace RecoLocalCalo_HGCalRecProducers {
```

Comment:
no need to instantiate objects in classes.h anymore, just the #include statements are needed